.github/workflows/generate_docs.yml

```diff
@@ -0,0 +1,44 @@
+name: generate docs for jsonschema
+on:
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      # Setup Python (faster than using Python container)
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+      # Setup node for jsonschema to markdown
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
+      # generate the markdown from json schema
+      # the 'sed' command removes a link placed by the tool that conflicts with jekyll preferences
+      - name: generate block schema doc
+        run: |
+          rm -rf ./docs/blocks
+          mkdir ./docs/blocks
+          echo 
```

Comment:
what's the purpose of it?

Reply:
re-create the blocks doc folder. so that if a block was removed you don't end up with leftovers